package dev.vality.fraudbusters.notificator.exception;

public class WarehouseQueryException extends RuntimeException {

    public WarehouseQueryException(Throwable cause) {
        super(cause);
    }
}
